provisioning/etcd/README.md

```diff
@@ -0,0 +1,51 @@
+# Templates API Etcd Cluster
+
+## Overview
+
+- Peers and clients communicate using the unencrypted HTTP/gRPC protocol.
+- Communication is limited to participants using `NetworkPolicy`.
+  - Client must have `templates-api-etcd-client: true` label.
+  -  Clients connect to `templates-api-etcd.default.svc.cluster.local:2379`.
+- Authentication is mandatory, there is only one user - `root`.
+  - Root password is random, it is stored in `templates-api-etcd` secret under `etcd-root-password` key.
+- Number of replicas is defined by `ETCD_REPLICA_COUNT` env variable (default `3`).
+- To re-generate `provisioning/templates/etcd/yaml` run `./generate.sh` and copy the file.
+- The number of nodes cannot be changed later (simple logic in `build.sh` is not enough).
+
+## CLI Client For Testing
+
+Run container:
+```bash
+export ETCD_ROOT_PASSWORD=$(kubectl get secret --namespace default templates-api-etcd -o jsonpath="{.data.etcd-root-password}" | base64 -d)
+```
+
+```bash
+kubectl run -i -t templates-api-etcd-client --restart=Never --rm \
+  --namespace default \
+  --image docker.io/bitnami/etcd:3.5.4-debian-11-r27 \
+  --labels="templates-api-etcd-client=true" \
+  --env="ETCD_ROOT_PASSWORD=$ETCD_ROOT_PASSWORD" \
+  --env="ETCDCTL_ENDPOINTS="templates-api-etcd.default.svc.cluster.local:2379"" \
+  --command -- bash
+```
+
+Use client inside container:
+```bash
+etcdctl --user root:$ETCD_ROOT_PASSWORD put /message Hello
+etcdctl --user root:$ETCD_ROOT_PASSWORD get /message
+```
+
+## Local Testing / Deploy
```

Comment:
@webrouse Mimochodem, jak to zastavím? 🙂 Mám manuálně poshazovat pody nebo to jde nějak líp?

Reply:
tak asi sa to nejak ide pomocou `labels`, ... (s helmom staci `helm delete`) ... ja som si pri skusani zresetoval vzdy vsetko cez `minikube delete`.

Reply:
Jo, `minikube delete` je kůl. 🙂 Minikube jsem nepoužíval, tak v tom nemám přehled. A nechtěl jsem aby mi to tady zůstalo tiše běžet a požírat drahocený resoursy. 😄

Reply:
:) este by mohlo fungovat aj `kubectl delete all -l app=templates-api-etcd`

Reply:
A jo, jestli to má společná label, tak to je úplně ideální. Přidáš to prosím tě do toho readme?

Reply:
jj